test/integration/dataContract/validateDataContractFactory.spec.js

```diff
@@ -833,6 +833,52 @@ describe('validateDataContractFactory', () => {
         expect(error.keyword).to.equal('additionalProperties');
         expect(error.params.additionalProperty).to.equal('propertyNames');
       });
+
+      it('should have maxItems if uniqueItems is used', () => {
+        rawDataContract.documents.indexedDocument = {
+          type: 'object',
+          properties: {
+            something: {
+              type: 'array',
+              uniqueItems: true,
+            },
+          },
+          additionalProperties: false,
+        };
+
+        const result = validateDataContract(rawDataContract);
+
+        expectJsonSchemaError(result);
+
+        const [error] = result.getErrors();
+
+        expect(error.dataPath).to.equal('.documents[\'indexedDocument\'].properties[\'something\']');
+        expect(error.keyword).to.equal('required');
+        expect(error.params.missingProperty).to.equal('maxItems');
+      });
+
+      it('should have maxItems no bigger than 1000 if uniqueItems is used', () => {
```

Comment:
Just noticed this text is off by 2 orders of magnitude as compared with the [implementation above](https://github.com/dashevo/js-dpp/pull/108/files#diff-a4b7db40173a72aa559fe8563893e560R208)
```suggestion
      it('should have maxItems no bigger than 100000 if uniqueItems is used', () => {
```

Reply:
Fixed it. Thank you!